scripts/src/main/resources/scripts/functions

```diff
@@ -567,12 +567,8 @@ function doDownloadInternal() {
     doEcho "Artifact already exists at ${target}\nTo force update please delete the file and run again."
     return 255
   fi
-  local curl_opt=""
-  if [ -n "${quiet}" ]
-  then
-    curl_opt="--silent"
-  fi
-  curl ${curl_opt} -fL "${url}" -o "${tmp_file}"
+  SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+  bash $SCRIPT_DIR/mcurl -o "${tmp_file}" "${url}"
```

Comment:
avoid exposing additional variables (see e.g. #5) and instead use `local` variables.
Further, IMHO the way you determine the script dir is not 100% reliable. Simply use `${DEVON_IDE_HOME}/scripts` instead what will work in all cases.